mobile-app/app/screens/AppNavigator/screens/Balances/components/CreateOrEditAddressLabelForm.tsx

```diff
@@ -92,31 +170,50 @@ export const CreateOrEditAddressLabelForm = memo(({ route, navigation }: Props):
         displayClearButton={labelInput !== '' && labelInput !== undefined}
         onChangeText={(text: string) => {
           setLabelInput(text)
-          validateInput(text)
+          validateLabelInput(text)
         }}
         onClearButtonPress={() => {
           setLabelInput('')
-          setInputErrorMessage('')
+          type === 'create' ? validateLabelInput('') : setLabelInputErrorMessage('')
         }}
-        placeholder={translate('components/CreateOrEditAddressLabelForm', `${typeof labelInput === 'string' ? labelInput : 'Address {{index}}'}`, { index })}
+        placeholder={translate('components/CreateOrEditAddressLabelForm', `${typeof labelInput === 'string' ? labelInput : 'Address label {{index}}'}`, { index })}
```

Comment:
We can switch this to "Please enter an address label". Same for Address field itself